project-base/storefront/components/Pages/Customer/EditProfile/BillingAddress.tsx

```diff
@@ -18,21 +17,8 @@ export const BillingAddress: FC = () => {
 
     const formProviderMethods = useFormContext<CustomerChangeProfileFormType>();
     const formMeta = useCustomerChangeProfileFormMeta(formProviderMethods);
-    const { setValue } = formProviderMethods;
-
     const countriesAsSelectOptions = useCountriesAsSelectOptions();
 
-    useEffect(() => {
-        if (countriesAsSelectOptions.length > 0) {
-            const userCountryValue = formProviderMethods.getValues(formMeta.fields.country.name);
-            const selectedCountry =
-                countriesAsSelectOptions.find((option) => option.value === userCountryValue.value) ||
-                countriesAsSelectOptions[0];
-
-            setValue(formMeta.fields.country.name, selectedCountry, { shouldValidate: true });
-        }
-    }, [countriesAsSelectOptions, formMeta.fields.country.name, formProviderMethods, setValue]);
-
```

Comment:
This change will remove pre-selecting of users country, is that intentional?

Reply:
Users must add their Billing address during registration, so there is no need for a pre-selecting option from the select box.

Reply:
Actually, users who log in via social networks don't have any info. So I fixed it.